src/IO/lbsLoadReaderVT.py

```diff
@@ -193,7 +193,10 @@ def json_reader(self, returned_dict: dict, file_name: str, phase_ids, node_id: i
 
         # validate schema
         if SchemaValidator().is_valid(schema_to_validate=decompressed_dict):
-            print(f"{bcolors.OK}[LoadReaderVT] Valid JSON schema in {file_name}{bcolors.END}")
+            print(bcolors.HEADER
```

Comment:
```Python
print(f"{bcolors.HEADER}[LoadReaderVT]{bcolors.END} Valid JSON schema in {file_name}")
```

Reply:
Last one I am changing.
After that please check out #132